Content.Server/Construction/Components/TabletopComputerBoardComponent.cs

```diff
@@ -0,0 +1,10 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Construction.Components;
+[RegisterComponent, ComponentProtoName("TabletopComputerBoardComponent")]
```

Comment:
```suggestion
[RegisterComponent]
```
How long ago was this code written?